RequestManagementSystem/ConfigTemplate.cfg

```diff
@@ -97,14 +105,22 @@ Agents
       }
     }
   }
+  ##END
+  ##BEGIN CleanReqDBAgent
   CleanReqDBAgent
   {
      PollingTime = 60
      ControlDirectory = control/RequestManagement/CleanReqDBAgent
-     DeleteGraceDays = 30
+     # How many days, until finished requests are deleted
+     DeleteGraceDays = 60
+     # How many requests are deleted per cycle
      DeleteLimit = 100
+     # If failed requests are deleted
      DeleteFailed = False
-     KickGraceHours = 2
-     KickLimit = 100
+     # How many hours a request can stay assigned
+     KickGraceHours = 1
+     # How many requests are kicked per cycle
+     KickLimit = 10000
```

Comment:
It was 100. Are you OK to make it 10000?

Reply:
This is what was in the code
https://github.com/DIRACGrid/DIRAC/blob/e80e60df88663927cb6b258920a86c1339a7b1f8/RequestManagementSystem/Agent/CleanReqDBAgent.py#L49-L50